dask/dask.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This initialization action script will install Dask and other relevant
+# libraries on a Dataproc cluster. This is supported for either "yarn" or
+# "standalone" runtimes Please see dask.org and yarn.dask.org for more
+# information.
+
+set -euxo pipefail
+
+readonly DEFAULT_CONDA_ENV=/opt/conda/miniconda3
+readonly DASK_YARN_CONFIG_DIR=/etc/dask/
+readonly DASK_YARN_CONFIG_FILE=${DASK_YARN_CONFIG_DIR}/config.yaml
+
+readonly DASK_RUNTIME="$(/usr/share/google/get_metadata_value attributes/dask-runtime || echo "yarn")"
+readonly RUN_WORKER_ON_MASTER="$(/usr/share/google/get_metadata_value attributes/dask-worker-on-master || echo "true")"
+
+readonly ROLE=$(/usr/share/google/get_metadata_value attributes/dataproc-role)
+readonly MASTER=$(/usr/share/google/get_metadata_value attributes/dataproc-master)
+
+# Dask 'standalone' config
+readonly DASK_LAUNCHER='/usr/local/bin/dask-launcher.sh'
+readonly DASK_SERVICE='dask-cluster'
+
+CONDA_PACKAGES=("dask")
+
+if [ "$(echo "$DATAPROC_VERSION < 2.0" | bc)" -eq 1 ]; then
+  CONDA_PACKAGES+=("fastavro" "fastparquet" "gcsfs")
+fi
```

Comment:
I think this is redundant, it's nice to have, but let's let users use `python` init action to install extra packages, we can document it in the readme.

Reply:
done